sdks/java/core/src/main/java/org/apache/beam/sdk/coders/SerializableCoder.java

```diff
@@ -57,6 +57,16 @@
     return new SerializableCoder<>(clazz, type);
   }
 
+  @Override
+  public boolean consistentWithEquals() {
+    return false;
+  }
+
+  @Override
+  public Object structuralValue(T value) {
```

Comment:
Can you please also add the documentation discussed on the thread - when this coder is used with objects with a bad `equals` method it is expected to get spurious mutations detected. I think there should be no other risk, as the only other use of `structuralValue` is in GBK, where a nondeterministic coder cannot be used anyhow.

Reply:
Updated